modules/bitgo/src/v2/coins/utxo/recovery/crossChainRecovery.ts

```diff
@@ -47,8 +46,25 @@ class BitgoPublicApi {
    * @returns {*}
    */
   async getUnspentInfo(addresses: string[]): Promise<Unspent[]> {
-    const url = this.coin.url(`/public/addressUnspents/${_.uniq(addresses).join(',')}`);
-    return (await request.get(url)).body as Unspent[];
+    const uniqueAddresses = new Set(addresses);
```

Comment:
Is there any merit to doing the one call first to the API and then attempting this secondary approach only if it is 404?

Reply:
yeah that's a good idea, i'll add that now. Thanks!